### ISSUE TITLE
Unable to move/copy/delete in SD card

### ISSUE THREAD

**Issue by Johnnyboyo14** [SimpleMobileTools/Simple-Gallery#1421]
Hello,
I just changed my SD card to a bigger 128gb one. Since the switch, I am unable to move, copy or delete any picture in my folders. Everything worked fine with the previous card.
I tried clearing cache, data, moving the app to the sd card and even reinstalling the app, as you suggested in other posts, but the problem still persists.
Any ideas ?
I'm running an unrooted android 9 phone.

**Comment by tibbi** [comment]
are you getting some error, or what exactly happens?

**Comment by Johnnyboyo14** [comment]
Yes I'm getting an "some files failed to be moved" messages at the bottom of my screen.
Also, when I try to create a new folder, I can't access my SD card using the app's file explorer.
I tried moving the pictures in the phones storage and everything works just fine.

Moving to the SD card works with other gallery apps and android's own file explorer.

Thank you for your help !

**Comment by braselectron** [comment]
I just moved to the latest commit because of similar problem.  After 44 photo folders it did not show new ones, nor show a folder after I move it to new place/rename.

After I update to latest commit I notice that I now need to use native files apk to create, rename or move folders or they will not show, in regard to SD card.

After new install I got a warning screen about using SD cards.

My phone is Samsung S7 SM-G930U Android 7.0 (I did not permit update from samsung - too many bad reviews).

Hope this feedback helps.
